src/jit/codegenxarch.cpp

```diff
@@ -1870,6 +1870,12 @@ void CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
             break;
 #endif // FEATURE_SIMD
 
+#if defined(_TARGET_XARCH_) && !defined(LEGACY_BACKEND)
```

Comment:
Perhaps, instead of having to modify this for each new target for which we add intrinsics, we should define a `FEATURE_HW_INTRINSICS`.

Reply:
Agree, will do.